packages/tos/src/TermsOfService.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Automattic\Jetpack;
+
+use Automattic\Jetpack\Connection\Manager;
+use Automattic\Jetpack\Status;
+
+class TermsOfService {
```

Comment:
Noting that we use underscores for separating words in class names, for example `JSON_Deflate_Array_Codec`.